src/sql/workbench/contrib/notebook/common/models/contracts.ts

```diff
@@ -9,7 +9,6 @@ export type CellType = 'code' | 'markdown' | 'raw';
 export class CellTypes {
 	public static readonly Code = 'code';
 	public static readonly Markdown = 'markdown';
-	public static readonly Raw = 'raw';
```

Comment:
any reason for removing this (besides the fact that we don't reference it probably 😄?)

Reply:
Only because it wasn't being used currently.